group/create.go

```diff
@@ -74,6 +79,18 @@ func GroupCreate(cmd *cobra.Command, args []string) error {
 		return fmt.Errorf("Creating Group: %w", err)
 	}
 
-	fmt.Printf("GroupID: %v\n", id)
+	if jsonOutput {
+		jsonId, err := json.MarshalIndent(
+			map[string]string{"id": id},
```

Comment:
Unfortunately for this one I can only output the ID because `helper.CreateGroup` returns `id` and not `group`. 
I could reproduce the code of `helper.CreateGroup` to get the `group` but I feel like it would be wasteful.
Also I could modify `helper` but that would cause problems for users expecting an `id`...

Reply:
Should I do the same for [`user/create.go`](https://github.com/passbolt/go-passbolt-cli/pull/7/files#diff-98ce049cb29b5ba4a77a499506eed149c34a0ef88fa0d53a312943c969d81209R80-R86) ?

Reply:
Yea, i think it would be best to return just the new ID for all create operations.